assets/constants.ts

```diff
@@ -5857,7 +5857,8 @@ export default {
     "translation", "voiceover", "question", "submit_question"
   ],
 
-  "QUESTIONS_SORT_KEY_DATE": "Date",
+  "SUGGESTIONS_SORT_KEY_DATE": "Date",
+  "SUGGESTIONS_SORT_KEY_NULL": "",
```

Comment:
Can we not have this constant? This is because Null/None should be different from empty string -- we should actually pass null / None instead.

Also, if possible, I think we shouldn't allow a case where the output is unsorted anyway. If we always require a sort order, it would make things simpler.